lib/Maintenance/Tasks/CleanupBrickTablesTask.php

```diff
@@ -27,11 +27,22 @@
  */
 class CleanupBrickTablesTask implements TaskInterface
 {
+    const PIMCORE_OBJECTBRICK_CLASS_DIRECTORY = PIMCORE_CLASS_DIRECTORY . '/DataObject/Objectbrick/Data';
+
     private LoggerInterface $logger;
+    private $mapLowerToCamelCase = [];
 
     public function __construct(LoggerInterface $logger)
     {
         $this->logger = $logger;
+        
+        $files = array_diff(scandir(self::PIMCORE_OBJECTBRICK_CLASS_DIRECTORY), array('..', '.'));
+        foreach ($files as $file) {
+            $classname = str_replace('.php','',$file);
+            $class = '\\Pimcore\\Model\\DataObject\\Objectbrick\\Data\\' . $classname;
+            $object = new $class(new \Pimcore\Model\DataObject\Concrete());
+            $this->mapLowerToCamelCase[strtolower($classname)] = $object->getType();
```

Comment:
```suggestion
            $classname = str_replace('.php','',$file);
            $this->mapLowerToCamelCase[strtolower($classname)] = $classname ;
```
I think the class name (file name) is identical to the type? Or am I missing something?